### PR TITLE
Update 05_Document_Thumbnails.md

### PR DESCRIPTION
**Pull request by kaurov**
Add documentation to use thumbnails for Assets of filetype document

### CODE REVIEW

File: doc/04_Assets/03_Working_with_Thumbnails/05_Document_Thumbnails.md [hunk 1]
```diff
@@ -31,4 +31,27 @@ if($asset instanceof Asset\Document) {
 ```
 
 > This feature requires Ghostscript and at least Gotenberg or LibreOffice be installed on the server.
+> To install it on Linux:
```
Comment (brusch) [inline doc/04_Assets/03_Working_with_Thumbnails/05_Document_Thumbnails.md]: Should we mention the possibility of Docker here as well?

Comment (kaurov) [reply doc/04_Assets/03_Working_with_Thumbnails/05_Document_Thumbnails.md]: Feel free

Comment (mcassier31) [inline doc/04_Assets/03_Working_with_Thumbnails/05_Document_Thumbnails.md]:
```suggestion
>
> Installation on Linux:
>
```

File: doc/04_Assets/03_Working_with_Thumbnails/05_Document_Thumbnails.md [hunk 2]
```diff
@@ -31,4 +31,27 @@ if($asset instanceof Asset\Document) {
...
+> sudo apt-get install ghostscript
```
Comment (mcassier31) [inline doc/04_Assets/03_Working_with_Thumbnails/05_Document_Thumbnails.md]:
```suggestion
> `sudo apt-get install ghostscript`
>
```

File: doc/04_Assets/03_Working_with_Thumbnails/05_Document_Thumbnails.md [hunk 3]
```diff
@@ -31,4 +31,27 @@ if($asset instanceof Asset\Document) {
...
+> sudo apt-get install libreoffice
```
Comment (mcassier31) [inline doc/04_Assets/03_Working_with_Thumbnails/05_Document_Thumbnails.md]:
```suggestion
> `sudo apt-get install libreoffice`
>
> You also have the possibility to install Gotenberg with a Docker image. To make it available, simply add the following config to your `docker-compose.yaml` file:
>
> ```yaml
>  services:
>     gotenberg:
>        image: gotenberg/gotenberg:7
>  ```
>
```

File: doc/04_Assets/03_Working_with_Thumbnails/05_Document_Thumbnails.md [hunk 4]
```diff
@@ -31,4 +31,27 @@ if($asset instanceof Asset\Document) {
...
+> 
+> To install it on Mac:
+> brew install ghostscript
```
Comment (mcassier31) [inline doc/04_Assets/03_Working_with_Thumbnails/05_Document_Thumbnails.md]:
```suggestion
>
> `brew install ghostscript`
>
```

File: doc/04_Assets/03_Working_with_Thumbnails/05_Document_Thumbnails.md [hunk 5]
```diff
@@ -31,4 +31,27 @@ if($asset instanceof Asset\Document) {
...
+> brew install --cask libreoffice
```
Comment (mcassier31) [inline doc/04_Assets/03_Working_with_Thumbnails/05_Document_Thumbnails.md]:
```suggestion
> `brew install --cask libreoffice`
```

File: doc/04_Assets/03_Working_with_Thumbnails/05_Document_Thumbnails.md [hunk 6]
```diff
@@ -31,4 +31,27 @@ if($asset instanceof Asset\Document) {
 ```
 
 > This feature requires Ghostscript and at least Gotenberg or LibreOffice be installed on the server.
+> To install it on Linux:
+> sudo apt-get install ghostscript
+> sudo apt-get install libreoffice
+> 
+> To install it on Mac:
```
Comment (mcassier31) [inline doc/04_Assets/03_Working_with_Thumbnails/05_Document_Thumbnails.md]:
```suggestion
> Installation on Mac:
```

**Review by mcassier31 — CHANGES_REQUESTED** [review]
Adds Docker installation and changes presentation for better readability.

**Review by mcassier31 — APPROVED** [review]

**Pull request merged**